.github/ISSUE_TEMPLATE/question.yaml

```diff
@@ -0,0 +1,15 @@
+name: 🙋 Question and Discussion
+description: Discuss or ask question about how to use prompttools and what the best practice may be.
```

Comment:
Type: ask questions on how to ...
Similar to the comment earlier. Which prompt tool are we referring to?